plugins/node/opentelemetry-instrumentation-pg/src/utils.ts

```diff
@@ -176,7 +212,10 @@ export function patchCallback(
         code: SpanStatusCode.ERROR,
         message: err.message,
       });
+    } else {
+      handleExecutionResult(instrumentationConfig, span, res);
```

Comment:
Shouldn't we also call the response hook in the [`patchCallbackPGPool` function below](https://github.com/open-telemetry/opentelemetry-js-contrib/pull/528/files#diff-7dcebb495f668f5ac147cb016849da23aa4df3c452932bfd9f2e95d9e8f3bc4cR240)?
Probably not, as I see there are tests for that, but how come it works?

Reply:
The PgPool instrumentation added an additional patching to the `pgPool.connect` method, and uses the same `query` patching.

Reply:
I have to admit I'm not very familiar with this instrumentation and its behavior.
So in what cases is "patchCallbackPGPool" being called?